models/articles.model.js

```diff
@@ -0,0 +1,23 @@
+const db = require('../db/connection')
+
+function selectArticle(article_id) {
+
+
+    return db.query(`SELECT * from articles where article_id =  $1`, [article_id])
+        .then((result) => {
+            const article = result.rows[0]
+            if (!article) {
+                return Promise.reject({
+                    custom_error: {
```

Comment:
Quite like your approach of putting it on a key of `custom_error` to make it clear what's happening